src/bin/annotate.pl

```diff
@@ -1162,7 +1065,7 @@ sub quantification {
 	my ($bp1, $bp2) = ($SV->{first_bp}, $SV->{second_bp});
 	my ($chr1, $pos1, $start1, $end1) = ($bp1->{tname}, $bp1->{tpos}, $bp1->{ort}, $bp1->{tstart}, $bp1->{tend});
 	my ($chr2, $pos2, $start2, $end2) = ($bp2->{tname}, $bp2->{tpos}, $bp2->{ort}, $bp2->{tstart}, $bp2->{tend});
-	$debug = 1 if(abs($pos1 - 170818803)<10 || abs($pos2 - 170818803)<10);
+	$debug = 0 if(abs($pos1 - 170818803)<10 || abs($pos2 - 170818803)<10);
```

Comment:
What is this doing?

Reply:
I actually didn't mean to update this (I did s/$debug = 1/$debug = 0/ to turn off debugging). It appears that Yongjin/Liqing must have been having an issue with this position and turned on debugging for all samples any time this position is encountered.